docs/glossary.md

```diff
@@ -155,14 +155,14 @@ Webpack
 Webpack entrypoint
     The main files generated by webpack as a result.
     They typically contain the application source code based on modules bundled together, but it can also include other resources, such as static resources.
-    It can contain code to automatically trigger the load of other JavaScript code files called "chunks".
+    They can contain code to automatically trigger the load of other JavaScript code files called "chunks".
 
 Babel
-    A JavaScript compiler that "transpiles" newer standards JavaScript to something that any browser can load.
+    A JavaScript compiler that "translates" newer standards JavaScript to something that any browser can load.
 
 Express
-    A JavaScript HTTP server with a simple API to build custom applications.
-    Volto uses it as its server.
+    A JavaScript HTTP server with a simple API, to build custom applications.
```

Comment:
We should instead use the official definition from [the source](https://expressjs.com/).

```suggestion
    [Express](https://expressjs.com/) is a minimal and flexible Node.js web application framework that provides a robust set of features for web and mobile applications.
```